lib/network/api/api.go

```diff
@@ -31,16 +33,22 @@ type NetworkHandlerAPI struct {
 	localNode *node.LocalNode
 	network   network.Network
 	storage   *storage.LevelDBBackend
+	version   string
 }
 
 func NewNetworkHandlerAPI(localNode *node.LocalNode, network network.Network, storage *storage.LevelDBBackend) *NetworkHandlerAPI {
 	return &NetworkHandlerAPI{
 		localNode: localNode,
 		network:   network,
 		storage:   storage,
+		version:   APIVersionV1,
```

Comment:
👍 I guess you can preparing for supporting multiple API environment.

Reply:
> I want to listen why not versioned in node handlers?

I think that node API has version prefix is good too. :-)